sysutils/diffutils/Portfile

```diff
@@ -62,6 +62,13 @@ configure.args      --program-prefix=g \
                     --infodir=${prefix}/share/info \
                     --mandir=${prefix}/share/man
 
+platform darwin 10 {
+# This is a fix for Rosetta
+    if {${build_arch} eq "ppc"} {
+    configure.args-append --build=powerpc-apple-darwin10
```

Comment:
This is nit-picky, but would you be willing to fix the indentation for this line?

Reply:
Now good?

Reply:
Yep, thanks for the quick fix Sergey!